### ISSUE TITLE
Errors when using `filter` on mobile

### ISSUE THREAD

**Issue by sheldonbaker** [d3/d3-brush#96]
On a mobile device, when using `brush.filter`, any events that are removed by the filter will end up causing an error of `emitter(...).moved is not a function` or `emitter(...).touchended is not a function`.

```
function touchmoved(event) {
  emitter(this, arguments).moved(event);
}

function touchended(event) {
  emitter(this, arguments).ended(event);
}
  ```

https://codepen.io/sheldoncodes/pen/jOxvmdq (simulate a mobile device in your devtools).

**Comment by muellnerlisa** [comment]
I am experiencing the exact same issue, did anyone find a solution for this error?

**Comment by sheldonbaker** [comment]
> I am experiencing the exact same issue, did anyone find a solution for this error?

No, I just had to filter the specific error message so it stays out of my logs.

**Comment by MathiasFranceschi** [comment]
Hi, i have the same issue when i try to filter the touchevents on conditions like if there are more than one finger on my device, i found too the same error here : https://observablehq.com/@d3/brush-filter when you emulate the mobile device and try to use touch events it send this error
